src/modules/endpoint/EndpointProvider.js

```diff
@@ -1,40 +0,0 @@
-import React from 'react'
-import PropTypes from 'prop-types'
-import Endpoint from './Endpoint'
-
-class EndpointProvider extends React.Component {
```

Comment:
How did you solve the mocking of endpoints without the EndpointProvider?

Reply:
I haven't tested the endpoints yet, so I didn't have to mock anything.